src/main/java/eu/darkbot/api/core/Tickable.java

```diff
@@ -0,0 +1,8 @@
+package eu.darkbot.api.core;
+
+public interface Tickable {
+
+    void onTick();
+
+    void onStopped();
+}
```

Comment:
I feel like tickable is useless and should be removed.

Module, Behaviour and Task should each implement their own (differently called) name for ticking, and only module and behaviour need an "onStopped" equivalent